auth0_flutter/test/web/auth0_flutter_web_test.dart

```diff
@@ -72,13 +74,91 @@ void main() {
 
   test('handleRedirectCallback is called on load when auth params exist in URL',
       () async {
+    final interop.RedirectLoginResult mockRedirectResult =
+        interop.RedirectLoginResult();
+
+    when(mockClientProxy.isAuthenticated())
+        .thenAnswer((final _) => Future.value(false));
+    when(mockClientProxy.handleRedirectCallback())
+        .thenAnswer((final _) => Future.value(mockRedirectResult));
+
+    plugin.urlSearchProvider = () => '?code=abc&state=123';
+    await auth0.onLoad();
+    verify(mockClientProxy.handleRedirectCallback());
+    verifyNever(mockClientProxy.checkSession());
+  });
+
+  test('handleRedirectCallback captures appState that was passed', () async {
+    final Map<String, Object?> appState = <String, Object?>{
+      'someFancyState': 'value',
+    };
+
+    final interop.RedirectLoginResult mockRedirectResult =
+        interop.RedirectLoginResult(
+      appState: appState,
+    );
+
     when(mockClientProxy.isAuthenticated())
         .thenAnswer((final _) => Future.value(false));
+    when(mockClientProxy.handleRedirectCallback(any))
+        .thenAnswer((final _) => Future.value(mockRedirectResult));
 
     plugin.urlSearchProvider = () => '?code=abc&state=123';
     await auth0.onLoad();
     verify(mockClientProxy.handleRedirectCallback());
     verifyNever(mockClientProxy.checkSession());
+
+    final Object? capturedAppState = await auth0.appState;
+
+    expect(capturedAppState, isNotNull);
+    expect(capturedAppState, isA<Map<Object?, Object?>>());
+    capturedAppState as Map<Object?, Object?>;
+    const MapEquality<Object?, Object?> eq = MapEquality<Object?, Object?>();
+
+    expect(eq.equals(capturedAppState, appState), isTrue);
+  });
+
+  test('appState getter throws UnimplementedError when not overridden',
```

Comment:
This test was added as well. While most of the coverage report was for files that I didn't change, the report did notify about two untested lines in the default implementation of the getter.

Reply:
It's still reporting these lines as uncovered
<img width="1137" alt="Screenshot 2025-02-10 at 21 57 43" src="https://github.com/user-attachments/assets/0fe5f59c-6324-42aa-93d1-a8e46921e9c7" />

Reply:
Hmm, I'll have a look at it!